game.py

```diff
@@ -0,0 +1,127 @@
+from card import Card
+from color import Color
+from player import Player
+import constants
+import random
+
+
+class Game(Player, Card):
```

Comment:
Why this inherits from  Play and Card???

Reply:
Actually no need to on this stage..